scm-ui/ui-webapp/public/locales/en/commons.json

```diff
@@ -125,5 +125,11 @@
   },
   "importLog": {
     "title": "Import Log"
+  },
+  "notifications": {
+    "toastTitle": "Notification",
```

Comment:
```suggestion
    "toastTitle": "System Notification",
```

Also this.

Reply:
Why system? What if we build a plugin, which allows a user sending a notification to another user?